website/www/site/content/en/roadmap/_index.md

```diff
@@ -46,21 +46,19 @@ For details see [Roadmap for multi-SDK efforts](/roadmap/connectors-multi-sdk/).
 
 ## Go SDK
 
-The Go SDK is the newest SDK, and is the first SDK built entirely on the
-portability framework. See the [Go SDK's Roadmap](/roadmap/go-sdk) if this piques your
-interest.
+The Go SDK is not actively being developed. See the [Go SDK's Roadmap](/roadmap/go-sdk)
```

Comment:
Can we say something like: `The Go SDK is not actively being developed beyond bugfixes due to lack of contributors. If you'd like to help move this forward again, see the [Go SDK's Roadmap](/roadmap/go-sdk)`? Basically, I'd like it to be clear that folks are welcome to contribute here, there just isn't much happening due to contributor interest/time.

Reply:
@damccorm Good point, I will update the description.